lib/src/main/java/adminlte/entity_list_table/communication/http/tables/columns/AbstractColumn.java

```diff
@@ -24,7 +23,15 @@ public String getFieldName() {
         return this.fieldName;
     }
 
+    public String getCsvCellContent(Object object) {
```

Comment:
Again, I'm not sure if it's good behavior. Why don't we use getObjectValue.toString()?

Reply:
Same reasoning, it's more flexibility. For example, the table timestamp values might be processed to show localized times, while csv exports with the timestamp as written in the object